src/Survival.jl

```diff
@@ -13,12 +14,26 @@ export
 
     KaplanMeier,
     fit,
-    confint
+    confint,
+
+    coxph,
+    coef,
+    loglikelihood,
+    nullloglikelihood,
+    nobs,
+    dof,
+    vcov,
+    stderr
+
 
 abstract type AbstractEstimator end
 abstract type NonparametricEstimator <: AbstractEstimator end
 
 include("eventtimes.jl")
 include("kaplanmeier.jl")
+include("cox_utils.jl")
```

Comment:
@ararslan will tell, but why not group all three Cox-related files in a single file? Doesn't sound too big. That will keep the package cleaner when more models will be supported.

Reply:
Yeah, I like the idea of the Cox stuff grouped as a single file.